ui/components/ui/box/box.js

```diff
@@ -18,64 +18,85 @@ import {
 
 const BASE_CLASS_NAME = 'box';
 const Sizes = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
-export const BackgroundColors = [
-  COLORS.BACKGROUND_DEFAULT,
-  COLORS.BACKGROUND_ALTERNATIVE,
-  COLORS.OVERLAY_DEFAULT,
-  COLORS.PRIMARY_DEFAULT,
-  COLORS.PRIMARY_ALTERNATIVE,
-  COLORS.PRIMARY_MUTED,
-  COLORS.ERROR_DEFAULT,
-  COLORS.ERROR_ALTERNATIVE,
-  COLORS.ERROR_MUTED,
-  COLORS.WARNING_DEFAULT,
-  COLORS.WARNING_ALTERNATIVE,
-  COLORS.WARNING_MUTED,
-  COLORS.SUCCESS_DEFAULT,
-  COLORS.SUCCESS_ALTERNATIVE,
-  COLORS.SUCCESS_MUTED,
-  COLORS.INFO_DEFAULT,
-  COLORS.INFO_ALTERNATIVE,
-  COLORS.INFO_MUTED,
-  COLORS.MAINNET,
-  COLORS.ROPSTEN,
-  COLORS.KOVAN,
-  COLORS.RINKEBY,
-  COLORS.GOERLI,
-  COLORS.TRANSPARENT,
-  COLORS.LOCALHOST,
-];
-export const BorderColors = [
-  COLORS.BORDER_DEFAULT,
-  COLORS.BORDER_MUTED,
-  COLORS.PRIMARY_DEFAULT,
-  COLORS.PRIMARY_ALTERNATIVE,
-  COLORS.PRIMARY_MUTED,
-  COLORS.ERROR_DEFAULT,
-  COLORS.ERROR_ALTERNATIVE,
-  COLORS.ERROR_MUTED,
-  COLORS.WARNING_DEFAULT,
-  COLORS.WARNING_ALTERNATIVE,
-  COLORS.WARNING_MUTED,
-  COLORS.SUCCESS_DEFAULT,
-  COLORS.SUCCESS_ALTERNATIVE,
-  COLORS.SUCCESS_MUTED,
-  COLORS.INFO_DEFAULT,
-  COLORS.INFO_ALTERNATIVE,
-  COLORS.INFO_MUTED,
-  COLORS.MAINNET,
-  COLORS.ROPSTEN,
-  COLORS.KOVAN,
-  COLORS.RINKEBY,
-  COLORS.GOERLI,
-  COLORS.TRANSPARENT,
-  COLORS.LOCALHOST,
-];
+export const BACKGROUND_COLORS = pick(COLORS, [
+  'BACKGROUND_DEFAULT',
+  'BACKGROUND_ALTERNATIVE',
+  'OVERLAY_DEFAULT',
+  'PRIMARY_DEFAULT',
+  'PRIMARY_ALTERNATIVE',
+  'PRIMARY_MUTED',
+  'ERROR_DEFAULT',
+  'ERROR_ALTERNATIVE',
+  'ERROR_MUTED',
+  'WARNING_DEFAULT',
+  'WARNING_ALTERNATIVE',
+  'WARNING_MUTED',
+  'SUCCESS_DEFAULT',
+  'SUCCESS_ALTERNATIVE',
+  'SUCCESS_MUTED',
+  'INFO_DEFAULT',
+  'INFO_ALTERNATIVE',
+  'INFO_MUTED',
+  'MAINNET',
+  'ROPSTEN',
+  'KOVAN',
+  'RINKEBY',
+  'GOERLI',
+  'TRANSPARENT',
+  'LOCALHOST',
+]);
+
+export const BORDER_COLORS = pick(COLORS, [
+  'BORDER_DEFAULT',
+  'BORDER_MUTED',
+  'PRIMARY_DEFAULT',
+  'PRIMARY_ALTERNATIVE',
+  'PRIMARY_MUTED',
+  'ERROR_DEFAULT',
+  'ERROR_ALTERNATIVE',
+  'ERROR_MUTED',
+  'WARNING_DEFAULT',
+  'WARNING_ALTERNATIVE',
+  'WARNING_MUTED',
+  'SUCCESS_DEFAULT',
+  'SUCCESS_ALTERNATIVE',
+  'SUCCESS_MUTED',
+  'INFO_DEFAULT',
+  'INFO_ALTERNATIVE',
+  'INFO_MUTED',
+  'MAINNET',
+  'ROPSTEN',
+  'KOVAN',
+  'RINKEBY',
+  'GOERLI',
+  'TRANSPARENT',
+  'LOCALHOST',
+]);
 
+export const TEXT_COLORS = pick(COLORS, [
+  'TEXT_DEFAULT',
+  'TEXT_ALTERNATIVE',
+  'TEXT_MUTED',
+  'OVERLAY_INVERSE',
+  'PRIMARY_DEFAULT',
+  'PRIMARY_INVERSE',
+  'SECONDARY_DEFAULT',
+  'SECONDARY_INVERSE',
+  'ERROR_DEFAULT',
+  'ERROR_INVERSE',
+  'SUCCESS_DEFAULT',
+  'SUCCESS_INVERSE',
+  'WARNING_INVERSE',
+  'INFO_DEFAULT',
+  'INFO_INVERSE',
+]);
```

Comment:
These should be in design-system.js versus the Box component

Reply:
Yeah, it should be. I have added it here since we were using it only in Box for now. But now I have moved it to `design-systems`